examples/checkout/sdk-switch-network-with-nextjs/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
+}
```

Comment:
a root: true key value should be added to the config to ensure it's running eslint independently to the sdk repo's one